fvm/context.go

```diff
@@ -32,7 +31,6 @@ type Context struct {
 	ServiceEventCollectionEnabled bool
 	AccountFreezeAvailable        bool
 	ExtensiveTracing              bool
-	SignatureVerifier             crypto.SignatureVerifier
```

Comment:
this change might need some changes on the emulator and other places that has their own context and environments.

Reply:
I already checked the emulator doesn't use the interface. If it does, I'll also update the emulator. 
Is there another tool that you think uses its own context?